test/parallel/test-http-dns-error.js

```diff
@@ -29,7 +29,7 @@ const assert = require('assert');
 const http = require('http');
 const https = require('https');
 
-const host = '*'.repeat(256);
+const host = '*'.repeat(250);
```

Comment:
I know you were doing this just to see if the OOM error went away on SmartOS, but I think 256 was chosen specifically because 255 is the maximum length for a hostname. I believe the idea here is to trigger a hostname-too-long error specifically. So this change should not land, I suspect.